static/src/js/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -16,28 +17,77 @@ import Row from 'react-bootstrap/Row'
  */
 export const ErrorBanner = ({
   dataTestId,
-  message
-}) => (
-  <Row>
-    <Col>
-      <Alert variant="danger">
-        <Alert.Heading>Sorry!</Alert.Heading>
+  message,
+  previousURL
+}) => {
+  const [hasCMRLagError, setHasCMRLagError] = useState(false)
+  const [conceptIdDoesNotExist, setConceptIdDoesNotExist] = useState(false)
+  const currentURL = window.location.pathname
+  const conceptKeywords = ['/collections/', '/variables/', '/services/', '/tools/']
+
+  // Catches if we are coming from a published record, indicating this is a CMR lag error
+  useEffect(() => {
+    if (previousURL && conceptKeywords.some((keyword) => previousURL.includes(keyword))) {
+      setHasCMRLagError(true)
+    }
 
-        <span className="visually-hidden">{' '}</span>
+    if (currentURL && conceptKeywords.some((keyword) => currentURL.includes(keyword))) {
```

Comment:
do you need currentURL here?

Reply:
Thinking about this some more.   I don't think you need a state variable for this second case.  I think maybe helper function, something like isConceptUrl() and then use that helper in the "else" part of your render.